cluster/charts/crossplane/templates/crds/workload.crossplane.io_kubernetesapplicationresources.yaml

```diff
@@ -22,7 +22,9 @@ spec:
   group: workload.crossplane.io
   names:
     kind: KubernetesApplicationResource
+    listKind: KubernetesApplicationResourceList
```

Comment:
that's odd that these workload CRDs didn't seem to get these listKind/singular additions that all the other CRDs got in v0.3.  Looks like we may have had the wrong directory name in the controller-gen calls in our Makefile?

Reply:
Yes, that is the case unfortunately. Indeed, we kept registering `Policy` types as well since the generation did not remove anything but only add/replace.